submissions/modding_inspiration/modding_inspiration.md

```diff
@@ -0,0 +1,7 @@
+## Modding Inspiration
+
+I love getting feedback on my mods. Sometimes inspiration for new features are found in the oddest places. Last month, a comment was left on my [Generic Logistic Chest](https://mods.factorio.com/mod/Generic_Logistic_Chest) mod from a German YouTube channel linking to a spotlight of the mod. Not speaking any German myself, I still watched it a bit to see how the reviewer was showcasing the mod.
+
+The quick gist of the mod is that there are "generic" versions of logistic chests, that when placed opens a UI to change it to a normal logistic chest (storage, requester, etc). I saw the reviewer place a few of the chests, then they selected a chest from my UI. They appeared to expect that it would change all the chests. At that time, it only changed the last one placed by the player. It was just how it always worked, it wasn't a conscious decision, it just never occurred to me to let the player place several and then change all of them with one click.
```

Comment:
Could you add a bit more context about what the advantages of using the generic logistic chests are? Why was it you made the mod in the first place? Does this new feature change how you think about the mod in anyway?